docs/user/dashboard/tutorials.asciidoc

```diff
@@ -1,36 +1,79 @@
-[[vega-graph]]
-== Vega
+[[tutorials]]
+== Tutorials
```

Comment:
Why are there three timelion tutorials here?

Reply:
They were in the original docs that Alex wrote, so I just transferred them all over.